File: go/1115.go
package main

import "fmt"

func main() {
  var x, y int
  for {
    fmt.Scanf("%d", &x)
    fmt.Scanf("%d", &y)

    if x == 0 || y == 0 {
      break
    }

    if x > 0 && y > 0 {
      fmt.Println("primeiro")
      continue
    }
    
    if x < 0 && y > 0 {
      fmt.Println("segundo")
      continue
    }

    if x < 0 && y < 0 {
      fmt.Println("terceiro")
      continue
    }

    fmt.Println("quarto")
  }

}

File: lua/1115.lua
while true do
    local x = io.read("*n")
    local y = io.read("*n")

    if x == 0 or y == 0 then
        break
    end

    if x > 0 and y > 0 then
        print("primeiro")
    elseif x < 0 and y > 0 then
        print("segundo")
    elseif x < 0 and y < 0 then
        print("terceiro")
    else
        print("quarto")
    end
end
